about.html

```diff
@@ -1 +1,135 @@
-<!-- Your markup here! -->
\ No newline at end of file
+	<!DOCTYPE html>
+	<html>
+
+		<head>
+		    <!-- Meta Stuff -->
+		  <meta charset="utf-8">
+
+		  <!-- Bootstap CSS -->
+		  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+
+		  <!-- Our CSS -->
+		  <link href="styles.css" rel="stylesheet">
+
+		</head>
+
+		<body>
+			<nav class="navbar navbar-default">
+				<div class="container-fluid padding-20">
+			    	<div class="navbar-header">
+			    		<a class="navbar-brand" href="#">
+			        		<img class = 'flag' alt="Brand" src="assets/danish_flag.gif" height="25" width="25">
+			      		</a>
+			     	</div>
+					<span class='logo-text'>Danebook</span>
+		    		<input type='text' class='searchbar' placeholder="Search for Users">
+		    		<span class='name'>Harry</span>
+		    	</div>
+			</nav>
+
```

Comment:
Just copy and paste the standard navbar from Bootstrap and delete the parts you don't want. So much easier than trying to make your own. http://getbootstrap.com/components/#navbar